BUILD.md

```diff
@@ -104,6 +104,17 @@ then you will need to choose the `libstdc++11` ABI:
    conan profile update settings.compiler.libcxx=libstdc++11 default
    ```
 
+**Set up Boost library to use `std::string_view`** <br><br>
+The below commands ensure inter-operability between `boost::string_view` and 
+`std::string_view` types.
+```
+conan profile update 'conf.tools.build:cxxflags+=["-DBOOST_BEAST_USE_STD_STRING_VIEW"]' default
+conan profile update 'env.CXXFLAGS="-DBOOST_BEAST_USE_STD_STRING_VIEW"' default
+conan profile show default # sanity check: displays the default profile
+```
+Note: If you have other flags in the `conf.tools.build` or `env.CXXFLAGS 
+section`, make sure to retain and append the existing flags. 
+
```

Comment:
Do not use `<br>` in the text. Just use a new line. None of this text needs to be bolded either. I feel like this section fits better with the other compiler flag fixes in Troubleshooting, but it's fine to leave it here.

---

Ensure inter-operability between `boost::string_view` and `std::string_view` types:

```
conan profile update 'conf.tools.build:cxxflags+=["-DBOOST_BEAST_USE_STD_STRING_VIEW"]' default
conan profile update 'env.CXXFLAGS="-DBOOST_BEAST_USE_STD_STRING_VIEW"' default
```

If you have other flags in the `conf.tools.build` or `env.CXXFLAGS` sections, make sure to retain the existing flags and append the new ones. You can check them with:

```
conan profile show default
```